### PR TITLE
[FIX] Hierarchical Event Descriptors (HED) page update

### PR DESCRIPTION
**Pull request by effigies**
Squashed #1592 without some merge noise that reverted unrelated changes.

@tpatpa @VisLab Can you please review to make sure I didn't lose any of the originally planned changes?

Closes #1592.

### CODE REVIEW

**Review by dorahermes — APPROVED** [review]

File: src/appendices/hed.md [hunk 1]
```diff
@@ -106,18 +106,20 @@ The `duration` column is also annotated as a value column.
...
+of that row, as described in the next section.
 
-Example: The fully assembled annotation for the first event in the above
+**Example:** The fully assembled annotation for the first event in the above
 `events.tsv` file with onset `1.2` (the first row) is:
 
 ```Text
-Duration/0.6 s, Sensory-event, Visual-presentation, (Square, Blue),
-(Delay/1.435 ms, Agent-action, (Experiment-participant, (Press, Mouse-button))),
+(Duration/0.6 s, Sensory-event, Visual-presentation,
```
Comment (VisLab) [inline src/appendices/hed.md:115]: This is not correct without the curly brace notation, which is not yet supported by the HED javascript validator used in bids.  Until that all goes through, the annotation for the presentation cannot be enclosed in parentheses:

```code
Duration/0.6 s, Sensory-event, Visual-presentation, 
((Square, Red), (Computer-screen, Center-of)),
```

@effigies can you correct.  Tal will not be available for a while.

File: src/appendices/hed.md [hunk 2]
```diff
@@ -162,41 +164,57 @@ The HED vocabulary is specified by a HED schema,
...
-`HEDVersion` field be included when using HED in a BIDS dataset.
+The BIDS validator will generate an error if your dataset uses HED
+and the `HEDVersion` field is missing from the dataset description file.
+To avoid this, include a `HEDVersion` field in the `dataset_description.json`
+if you are using HED annotations.
 
 ### Using HED library schemas
 
-HED also allows you to use one or more specialized vocabularies along with or instead of
+HED also allows you to use one or more specialized vocabulary along with or instead of
```
Comment (VisLab) [inline src/appendices/hed.md:191]: I think this should be vocabularies not vocabulary  because of the "more".

File: src/appendices/hed.md [hunk 3]
```diff
@@ -162,41 +164,57 @@ The HED vocabulary is specified by a HED schema,
...
+A library schema is specified in the form `<library-name<_>library-version>`.
+
+#### Partnered library schemas
+
+Partnered library schemas were introduced in HED specification version 3.2.0
+and are supported by HED standard schema versions ≥ 8.2.0.
+A partnered library schema is tied to a specific version of
+the HED standard schema as specified in its header. A given library schema
+version is either partnered or standalone.
+
```
Comment (VisLab) [inline src/appendices/hed.md:204]: Add:

> A partnered schema is one whose vocabulary trees are merged with its standard schema partner when the
> schema is released. Thus, the two vocabularies appear as one vocabulary to the annotator.

File: src/appendices/hed.md [hunk 4]
```diff
@@ -206,22 +224,24 @@ SCORE library for clinical neurological annotation located at
   "HEDVersion": ["8.1.0", "sc:score_1.0.0"]
 }
 ```
-The `sc:` is a user-chosen prefixes used to distinguish the sources
+The `sc:` is a user-chosen prefix used to distinguish the sources
```
Comment (VisLab) [inline src/appendices/hed.md:227]: The `sc:` is a user-chosen prefix used to distinguish the source schemas

**Review by VisLab — CHANGES_REQUESTED** [review]
I've requested a few minor corrections. Thx!

**Comment by effigies** [comment]
@VisLab Updated with your suggestions (or minor variations thereon). Please LMK what you think.

**Review by VisLab — APPROVED** [review]
LGTM

**Pull request merged**